modules/ossm-migrating-read-me-tls-configuration-change.adoc

```diff
@@ -0,0 +1,23 @@
+
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/about/ossm-migrating-assembly.adoc
+
+//Start of an overall Migrating section.
+//Section is most likely to be reworked/restructured with OSSM 2 to OSSM 3 migration guides for GA. Unknown how many migration guides there are at this time (11/11/2024). It would be beneficial to be able to link from differences to the relevent migration guide so that users A) understand the change, esp significant changes like new operator, installing tracing and Kiali separately, gateways, etc.
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-migrating-read-me-transport-layer-security-configuration-change_{context}"]
+= Transport layer security (TLS) configuration change
+
+In {SMProductName} 2, you created the `ServiceMeshControlPlane` resource, and enabled mTLS strict mode by setting `spec.security.dataPlane.mtls` to `true`.
+
+You were able to set the minimum and maximum TLS protocol versions by setting the `spec.security.controlPlane.tls.minProtocolVersion` or `spec.security.controlPlane.tls.maxProtocolVersion` in your `ServiceMeshControlPlane` resource.
+
+In {SMProduct} 3, the `Istio` resource replaces the `ServiceMeshControlPlane` resource and does not include these settings.
+
+To enable enable mTLS strict mode in {SMProduct} 3, you need to apply the corresponding `PeerAuthentication` and `DestinationRule` resources.
+
+The TLS protocol version can be set through "Istio Workload Minimum TLS Version Configuration."
```

Comment:
Perhaps worth noting that this is now found at `spec.meshConfig.tlsDefaults.minProtocolVersion` in the `Istio` resource since we note where it was found in the SMCP.